mindsdb/integrations/handlers/flaml_handler/example.md

```diff
@@ -0,0 +1,84 @@
+| age  | sex  | cp   | trestbps | chol  | fbs  | restecg | thalach | exang | oldpeak | slope | ca   | thal | target |
++------+------+------+----------+-------+------+---------+---------+-------+---------+-------+------+------+--------+
+| 63.0 |  1.0 |  3.0 |    145.0 | 233.0 |  1.0 |     0.0 |   150.0 |   0.0 |     2.3 |   0.0 |  0.0 |  1.0 |    1.0 |
+| 37.0 |  1.0 |  2.0 |    130.0 | 250.0 |  0.0 |     1.0 |   187.0 |   0.0 |     3.5 |   0.0 |  0.0 |  2.0 |    1.0 |
+| 56.0 |  1.0 |  1.0 |    120.0 | 236.0 |  0.0 |     1.0 |   178.0 |   0.0 |     0.8 |   2.0 |  0.0 |  2.0 |    1.0 |
+| 57.0 |  0.0 |  0.0 |    120.0 | 354.0 |  0.0 |     1.0 |   163.0 |   1.0 |     0.6 |   2.0 |  0.0 |  2.0 |    1.0 |
+| 57.0 |  1.0 |  0.0 |    140.0 | 192.0 |  0.0 |     1.0 |   148.0 |   0.0 |     0.4 |   1.0 |  0.0 |  1.0 |    1.0 |
++------+------+------+----------+-------+------+---------+---------+-------+---------+-------+------+------+--------+
+```
+2. Create an AI Table with TPOT as the ML backend:
+   
+```sql
+CREATE MODEL 
+  mindsdb.heart_disease_flaml_model
+FROM files
+  (SELECT * FROM heart_disease )
+PREDICT target
+USING
+engine='FLAML';
+```
+
+
+
+
+3. Check the status of the predictor. It may take a while to finish training:
+
+```sql
+SELECT * FROM mindsdb.models where name='heart_disease_flaml_model';
+```
+```sql
++-------------+--------+---------+---------+----------+----------+---------+---------------+-----------------+-------+---------------------+-----------------------------------+------------------------+-----------------------+---------------------+------+----------------------------+
+| NAME        | ENGINE | PROJECT | VERSION | STATUS   | ACCURACY | PREDICT | UPDATE_STATUS | MINDSDB_VERSION | ERROR | SELECT_DATA_QUERY   | TRAINING_OPTIONS                  | CURRENT_TRAINING_PHASE | TOTAL_TRAINING_PHASES | TRAINING_PHASE_NAME | TAG  | CREATED_AT                 |
++-------------+--------+---------+---------+----------+----------+---------+---------------+-----------------+-------+---------------------+-----------------------------------+------------------------+-----------------------+---------------------+------+----------------------------+
+| heart_disease_flaml_model | FLAML  | mindsdb |       1 | complete | NULL     | target  | up_to_date    | 23.4.4.4        | NULL  | SELECT * FROM Heart | {'target': 'target', 'using': {}} | NULL                   | NULL                  | NULL                | NULL | 2023-05-07 10:53:46.937682 |
++-------------+--------+---------+---------+----------+----------+---------+---------------+-----------------+-------+---------------------+-----------------------------------+------------------------+-----------------------+---------------------+------+----------------------------+
+```
+
+3. Let me make a prediction for you. 
+_NOTE: As of now, FLAML has a requirement that all input columns must be specified. If data is missing for any of the columns, an error will be triggered._
```

Comment:
Add this to the readme please, either in the use case section (question 2) or system specs (question 4).

Reply:
Done